pkg/services/grpcserver/interceptors/auth.go

```diff
@@ -101,6 +115,14 @@ func (a *Authenticator) getSignedInUser(ctx context.Context, token string) (*use
 		return nil, fmt.Errorf("service account is disabled")
 	}
 
+	if signedInUser.Permissions[signedInUser.OrgID] == nil {
```

Comment:
@ArturWierzbicki does this seem like enough? i'm still trying to spin up on the access control code, so it's unclear if we need to check access at this point. it seems like a lot of the access control service code depends on a HTTP `ReqContext`

Reply:
yep! one example of how its used: [toddtreece/grpc-server-context](https://github.com/grafana/grafana/tree/toddtreece/grpc-server-context)

Reply:
yep! one example of how its used: https://github.com/grafana/grafana/blob/ebcdf402b2e0f5ca7e41382d13c6f04fe2824e23/pkg/services/apikey/apikeyimpl/sqlx_store.go#L37